ml_engine/online_prediction/predict.py

```diff
@@ -0,0 +1,184 @@
+#!/bin/python
+# Copyright 2017 Google Inc. All Rights Reserved. Licensed under the Apache
+# License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License. You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+"""Examples of using the Cloud ML Engine's online prediction service."""
+from __future__ import print_function
+
+import argparse
+import base64
+import json
+
+# [START import_libraries]
+import googleapiclient.discovery
+# [END import_libraries]
+
+
+# [START predict_json]
+def predict_json(project, model, instances, version=None):
+    """Send json data to a deployed model for prediction.
+
+    Args:
+        project (str): project where the Cloud ML Engine Model is deployed.
+        model (str): model name.
+        instances ([Mapping[str: any]]): dictionaries from string keys
+            defined by the model deployment, to data with types that match
+            expected tensors
```

Comment:
Yeah it doesn't make much sense with context. But could also use some rewording.